src/sentry/replays/post_process.py

```diff
@@ -116,7 +118,7 @@ def dict_unique_list(items: Iterable[tuple[str, str]]) -> dict[str, list[str]]:
 
 
 def _archived_row(replay_id: str, project_id: int) -> dict[str, Any]:
-    return {
+    archived_obj = {
```

Comment:
Additionally in Python its convention to spell out variable names explicitly.  So `archive_object` would be better.  But a name that explicitly defines what the object _is_ such as "row" or "response content" is best.  However you can think to name it.